cosmic-applet-time/src/window.rs

```diff
@@ -410,18 +540,3 @@ fn date_button(day: u32, is_month: bool, is_day: bool) -> Button<'static, Messag
         button
     }
 }
-
-fn time_subscription(update_at: Every) -> Subscription<()> {
```

Comment:
Why remove this subscription? I believe `duration_trunc` is important here to ensure that the subscription doesn't fall out of sync, and sends the event every minute or second reliably.

Reply:
@wiiznokes I saw you requested a review from me about 20 minutes ago. Did you see this comment?

Reply:
Oops no i didn't, sorry

Reply:
I initially removed it because it didn't worked well with show second, but i will revert that